user_guide/data_management.rst

```diff
@@ -0,0 +1,234 @@
+.. _user-guide-data-management-basics:
+
+Data Management Basics
+======================
+
+With OroCRM, you can record, store, process and analyze various customer-related information. This information may cover 
+different areas including personal and contact details, records of the customer activity, possible sales and 
+arrangements, and many others. This can be any type of information necessary to monitor, manage and understand specific 
+customer-related activities. 
```

Comment:
Edit: With OroCRM, you can record, store, process, and analyze various customer-related data. This information may cover different areas, including personal and contact details, records of the customer activity, possible sales and arrangements, and many others. This can include any type of information necessary to monitor, manage, and understand specific customer-related activities.